cmd/runner/main.go

```diff
@@ -44,9 +45,23 @@ func main() {
 	cfg := makeRunnerConfig()
 	fmt.Println(cfg)
 
-	rec := request.Do(cfg)
+	r := requester.New(requester.Options{
+		Concurrency: cfg.RunnerOptions.Concurrency,
+		Requests:    cfg.RunnerOptions.Requests,
+		Duration:    cfg.RunnerOptions.GlobalTimeout,
+		Timeout:     cfg.Request.Timeout,
+	})
```

Comment:
`config.Config` was not structured in a way that allowed the definition of `Requester` as a _doer_.
This interfacing is temporary. We need to discuss how to do this cleanly, suggestions more than welcomed.